deploy/kubernetes/fluentd-sumologic.yaml.tmpl

```diff
@@ -57,55 +54,87 @@ data:
         @id sumologic.endpoint.metrics.apiserver
         endpoint "#{ENV['SUMO_ENDPOINT_METRICS_APISERVER']}"
         @include metrics.output.conf
+        <buffer>
+          @type memory
```

Comment:
@vsinghal13 is this change is only for helm based installation? For non-helm, we are still going to buffer to memory?

Reply:
I think the non-helm users can customize config by following these steps: https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/master/deploy/docs/Non_Helm_Installation.md#customize-configuration